packages/paste-website/src/styles/docSearch.ts

```diff
@@ -1,161 +1,112 @@
 import {css} from '@twilio-paste/styling-library';
+import type {CSSObject} from '@twilio-paste/styling-library';
+import {pasteBaseStyles} from '@twilio-paste/theme';
+import type {GenericThemeShape} from '@twilio-paste/theme';
 
+export const docSearchVariable = (theme: GenericThemeShape): CSSObject => {
+  return {
+    ':root': {
+      '--docsearch-primary-color': theme.backgroundColors.colorBackgroundPrimaryStronger,
+      '--docsearch-text-color': theme.textColors.colorText,
+      '--docsearch-spacing': theme.space.space60,
```

Comment:
Change to space50 so the search icon in the modal isn't so far away from the border.

As far as I can tell, the other stuff that this variable touches is ok if the spacing is bumped down.